src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -4015,18 +4015,18 @@
      * </pre>
      * <p>parses as:</p>
      * <pre>
-     * LITERAL_IF (if)
-     *  |--LPAREN (()
-     *  |--EXPR
-     *  |   `--LITERAL_INSTANCEOF (instanceof)
-     *  |       |--IDENT (obj)
-     *  |       `--PATTERN_VARIABLE_DEF
-     *  |            |--TYPE
-     *  |            |   `--IDENT (String)
-     *  |            `--IDENT (str)
-     *  |--RPAREN ())
-     *  `--SLIST ({)
-     *      `--RCURLY (})
+     * --LITERAL_IF -&gt; if
```

Comment:
Remove leading "--" to indent whole block to left

Reply:
I have made the requested changes.